app/code/Magento/Store/Model/StoreCreateManagement.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Magento\Store\Model;
```

Comment:
Please add the header to all the classes
```suggestion
<?php
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
declare(strict_types=1);

namespace Magento\Store\Model;
```